ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -38,13 +38,15 @@
   public final Histogram selectorSelectTime;
   public final Counter selectorIORate;
   public final Histogram selectorIOTime;
+  public final Histogram selectorPerceivedSslHandshakeTime;
   public final Counter selectorNioCloseErrorCount;
   public final Counter selectorDisconnectedErrorCount;
   public final Counter selectorIOErrorCount;
   public final Counter selectorKeyOperationErrorCount;
   public final Counter selectorCloseKeyErrorCount;
   public final Counter selectorCloseSocketErrorCount;
   public Gauge<Long> selectorActiveConnections;
+  public Gauge<Long> selectorConnectionsPendingHandshake;
```

Comment:
these two variables could be named `selectorNumActiveConnections` and `selectorNumConnectionsPendingHandshake` (since "count" seems to be used here for `Counter` only).